graphql-cli/src/main/java/io/ballerina/graphql/cmd/mappers/GraphqlConfig.java

```diff
@@ -0,0 +1,55 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.graphql.cmd.mappers;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Load the GraphQL config file as a custom class.
```

Comment:
POJO class representing contents of GraphQL config file

Reply:
This is resolved & changes are amended now.